client/src/components/types.ts

```diff
@@ -35,7 +29,35 @@ export enum RowType {
     Person,
 }
 
-export interface RowEvent {
-    type: RowType;
-    data: Object;
+export enum Events {
+    OverflowClick = 'overflowClick',
+    AcceptDocument = 'acceptDocument',
+    DeclineDocument = 'declineDocument',
+    Camera = 'toggleCamera',
+    SendDocument = 'sendDocument',
+    TerminateDocument = 'terminateDocument',
+    CreateInvitation = 'createInvitation',
+    RemoveInvitation = 'removeInvitation',
+    ShowUserInfo = 'showUserInfo',
+    DeleteUser = 'deleteUser',
+    EditUser = 'editUser',
 }
+
+export interface ContextPayload {
+    ty: RowType.AcceptDocument | RowType.Inbox;
+    id: number;
+}
+
+export interface InvitePayload {
+    ty: RowType.Invite;
+    email: string;
+    office: number;
+}
+
+export interface PersonPayload {
+    ty: RowType.Person;
+    id: number;
+    office: number;
+}
```

Comment:
Future PRs should consider importing the `model` types instead of repeating them here. However, this won't be possible until the CI for #6 and #30 passes. For now, we can approve this PR.